src/V3EmitCImp.cpp

```diff
@@ -289,52 +427,115 @@ class EmitCImp final : EmitCFunc {
             }
         }
     }
-    void emitAll(const AstNodeModule* modp) {
-        if (m_slow) {
-            // Emit static variable definitions
-            const string modName = prefixNameProtect(modp);
-            for (const AstNode* nodep = modp->stmtsp(); nodep; nodep = nodep->nextp()) {
-                if (const AstVar* const varp = VN_CAST_CONST(nodep, Var)) {
-                    if (varp->isStatic()) {
-                        puts(varp->vlArgType(true, false, false, modName));
-                        puts(";\n");
-                    }
+    void emitCommonImp(const AstNodeModule* modp) {
+
+        // Emit common implementation contents for given AstNodeModule
+        const auto emit = [this](const AstNodeModule* modp) {
```

Comment:
Can a new member function be used instead of a lambda?  Ok if not, but if not this function is getting long and perhaps should be split up.